### PR TITLE
Removes heist mode & associated code

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
This PR removes:
- The heist mode
- The `Vox Raider` role
- Assorted spawning for raiders (its so awful)
- Vox dartgun and spike thrower (The code for this makes me throw up)

Admins can still adminbus in raider teams with the `Spawn Gimmick Team` secret and use the `Vox Raider` outfit. There is also a shuttle template for the skipjack to load into the admin shuttle CC dock.

Leadership was consulted before making this PR, with general consensus being "remove it but allow admins to bus alternatives in" 

## Why It's Good For The Game
In a dataset of 7247 rounds, spanning 2018-03-21 23:55:05 to 2021-12-19 15:45:43, raiders/heist has been played 4 times, and one of those rounds lasted a mere 4 minutes (see column `dd` below)
image

The mode has no proper format, vague objectives with an "instant end" button, **terrible** code of the 2014-ponies era, and really has no place in modern day paradise.

## Images of changes
See MapDiffBot.
- Stations have had the skipjack docking ports removed
- CC has had the transit area and docking area removed, and a slight cleanup to the trader area

## Changelog
:cl:
del: Removed vox raiders. They were used 4 times in the past 3 (almost 4) years, and are broken as hell.
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
I mapmerged this

**Comment by S34NW** [comment]
since MDB is having a fit, here is the trader dock
image

**Comment by AffectedArc07** [comment]
> since MDB is having a fit, here is the trader dock image

Its because of your DMI messup
```
StdErr:

thread 'main' panicked at 'assertion failed: `(left == right)`
  left: `"version = "`,
 right: `"version = 4.0"`', src/dreammaker/dmi.rs:357:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

```

**Comment by S34NW** [comment]
Alright sassy pants

**Comment by AffectedArc07** [comment]
Skipjack shuttle returns as a template for the admin shuttle dock at CC.

**Comment by matttheficus** [comment]
image

**Comment by Denthamos** [comment]
With the skipjack shuttle staying as a shuttle type for the admin shuttle variants and the vox raiders becoming a gimmick team for use in adminbus events, throwing my approval on this.

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -138,3 +138,8 @@
 	suffix = "armory"
 	name = "NRV Sparta"
 	description = "Armory Shuttle, with plenty of guns to hand out and some general supplies."
+
+/datum/map_template/shuttle/admin/hospital
```
Comment (dearmochi) [inline code/datums/shuttles.dm:142]: Might've forgotten to rename the typepath.

**Review by dearmochi — CHANGES_REQUESTED** [review]

File: code/datums/shuttles.dm (hunk 1 above)
Comment (AffectedArc07) [reply code/datums/shuttles.dm:142]: Im really slipping

Comment (S34NW) [reply code/datums/shuttles.dm:142]: its not just me!

**Review by dearmochi — APPROVED** [review]

File: code/datums/shuttles.dm (hunk 1 above)
Comment (AffectedArc07) [reply code/datums/shuttles.dm]: No this was tested, I just didn't realise a shuttle was missing from the list.

It didn't break the entire server.

**Pull request merged**